opensoundscape/localization.py

```diff
@@ -109,82 +99,61 @@ def estimate_location(
 
         Args:
             algorithm: 'gillette' or 'soundfinder', see localization.localize()
-            cc_threshold: see SpatialEvent documentation
+            cc_threshold: see SpatialEvent documentation. If None, uses self.cc_threshold value
+            cc_filter: filter for generalized cross correlation, see
+                opensoundscape.signal_processing.gcc()
             min_n_receivers: if number of receivers with cross correlation exceeding
-                the threshold is fewer than this, raises InsufficientReceiversError
+                the threshold is fewer than this, estimates location None
                 instead of estimating a spatial location
 
         Returns:
-            location estimate as cartesian coordinates (x,y) or (x,y,z) (units: meters)
-
-        Raises:
-            InsufficientReceiversError if the number of receivers with cross correlation
-                maximums exceeding `cc_threshold` is less than `min_n_receivers`
+            Location estimate as cartesian coordinates (x,y) or (x,y,z) (units: meters)
 
         Effects:
             sets the value of self.location_estimate to the same value as the returned location
         """
         # if cc_threshold is None, use the value stored in the object
-        if cc_threshold is None:
+        if cc_threshold == "default":
```

Comment:
I think None is the typical/expected value of a default argument in Python when the user wants to not specify an argument and "let the function choose whats best"

Reply:
This is a problem I hit. We want the user to be able to specify "do not apply a cc_threshold". They can do so using `cc_threshold=None`. This is different from the user specifying "just use the default cc_threshold specified by the SpatialEvent.cc_threshold attribute". I think the way I solved this looks a bit wrong. Maybe specifying `cc_threshold=0` to mean do not apply a cc_threshold? However we hit the same problem again with `bandpass_range`